packages/create-waku-app/build.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+const path = require("path");
+const fs = require("fs-extra");
```

Comment:
I would avoid doing so since it slightly changes flow of execution and rules applied to the file: absence of global variables such as `__dirname` etc.

Reply:
IMO, we should release code with the latest syntax. 

Can you expand more on why you would lose access to global variables?

Reply:
I would refrain from using `ES modules`, at least for now. Historically it was meant to work with `require` and `imports` is creating friction for support of the script also I would consider it not latest but alternative way of writing JS in node, they are just different. 

As for `__dirname` here is an [example](https://stackoverflow.com/questions/46745014/alternative-for-dirname-in-node-js-when-using-es6-modules) of the issue (there are other workarounds) but it shows how limited/different are `ES modules` in Node.js comparing to common way.

Reply:
After a conversation with @weboko, it was concluded that sticking to `require` style syntax makes sense for the scope of this example.

However, an open conversation in re to using ES6 style imports/exports for consistency of Waku ecosystem remains.
Will open an issue as a followup to track the same.

Reply:
I was going to says "it's NodeJS so CJS is fine" but ESM is actually marked as stable in NodeJS 18 so I agree worth considering using ESM if it does not bring too much headache.

We do use a lot of experimental features for various tools in js-waku to make ESM work, I am happy to be more cautious when ESM is not needed (ie, here).

https://nodejs.org/dist/latest-v18.x/docs/api/esm.html